src/resources/items.py

```diff
@@ -0,0 +1,22 @@
+from .base import DungeonBase
+from .character import Character
+
+
+class DungeonItem(DungeonBase):
```

Comment:
This should be named `DungeonItems`

Reply:
or maybe `ItemManager`

You can put a draw function in here as well that draws all the items in the list.

Reply:
If this is a manager it doesn't need to extend DungeonBase